corehq/apps/geospatial/tasks.py

```diff
@@ -55,7 +57,12 @@ def index_es_docs_with_location_props(domain):
                 chunk_size=DEFAULT_CHUNK_SIZE,
             )
             celery_task_tracker.update_progress(current=i + 1, total=batch_count)
-    except Exception:
+    except Exception as e:
         celery_task_tracker.mark_as_error(error_slug='CELERY')
+        notify_exception(
+            None,
```

Comment:
nit: how do we ensure this is followed up on?

Reply:
The domain will receive an error alert if the Celery task fails, asking them to reach out to support. Sending this off to Sentry will allow us to have a bit more insight other than a generic `CELERY_ERROR` message to understand what could have gone wrong.

Reply:
Cool, that works :+1: